### DISCUSSION TITLE
Mirai登录问题

### DISCUSSION THREAD

**Discussion by star4277** [simple-robot/Discussions#121] [Q&A] [answered]
### 涉及的编程语言

Java

### 涉及的组件库

mirai (QQ)

### 疑问描述

除了通过.bot.*的配置文件查找类路径下的文件来启动bot外，还有没有其他方式启动bot，比如通过代码启动，或者将bot配置文件放在类路径之外读取来启动

**Comment by ForteScarlet** [comment, marked as answer]
类路径之外参考 
- https://github.com/simple-robot/simpler-robot/issues/503

代码启动方式参考
- https://github.com/orgs/simple-robot/discussions/81 中提到的 `动态注册BOT`
